projects/1.18/assets/daves-building-extended/davebuildingmod/lang/zh_cn.json

```
@@ -0,0 +1,705 @@
+	"gui.davebuildingmod.particle_generator_gui.label_insert_book":"插入",
+	"gui.davebuildingmod.particle_generator_gui.label_particle_generator_gui_wip":"粒子产生器GUI说明（制作中功能）",
+	"gui.davebuildingmod.particle_generator_gui.label_particlename":"粒子ID",
+	"gui.davebuildingmod.particle_generator_gui.label_particlename1":"粒子ID",
+	"gui.davebuildingmod.particle_generator_gui.label_select_amount":"选择数量",
+	"gui.davebuildingmod.particle_generator_gui.label_select_particle":"选择粒子",
+	"gui.davebuildingmod.particle_generator_gui.label_select_size":"选择尺寸",
+	"gui.davebuildingmod.particle_generator_gui.label_set_yoffset":"选择Y轴偏移",
+	"gui.davebuildingmod.particle_generator_gui.particle_amount_textfield":"粒子数量",
+	"gui.davebuildingmod.particle_generator_gui.particle_selection_textfield":"粒子名称",
+	"gui.davebuildingmod.particle_generator_gui.particle_size_textfield":"粒子区域",
+	"gui.davebuildingmod.particle_generator_gui.y_offset_textfield":"期望偏移量",
+	"gui.davebuildingmod.sound_generator_gui.button_load":"加载",
+	"gui.davebuildingmod.sound_generator_gui.button_set":"设定",
+	"gui.davebuildingmod.sound_generator_gui.label_particle_generator_gui_wip":"声音产生器GUI说明（制作中功能）",
+	"gui.davebuildingmod.sound_generator_gui.label_select_amount":"选择速度",
+	"gui.davebuildingmod.sound_generator_gui.label_select_particle":"选择模组ID",
+	"gui.davebuildingmod.sound_generator_gui.label_select_size":"选择尺寸",
+	"gui.davebuildingmod.sound_generator_gui.label_set_yoffset":"选择声音",
+	"gui.davebuildingmod.sound_generator_gui.particle_amount_textfield":"播放速度",
+	"gui.davebuildingmod.sound_generator_gui.particle_selection_textfield":"模组ID",
+	"gui.davebuildingmod.sound_generator_gui.particle_size_textfield":"声音区域",
+	"gui.davebuildingmod.sound_generator_gui.y_offset_textfield":"声音名称",
+	"item.buildingmod.advanced_circuit":"高级电路板",
+	"item.buildingmod.basic_circuit":"基础电路板",
+	"item.buildingmod.particle_card_fire":"粒子卡“火焰”",
+	"item.buildingmod.particle_card_tnt":"粒子卡“爆炸”",
+	"item.buildingmod.sound_card_computer_bleep":"声音卡“电脑哔哔声”",
+	"item.buildingmod.sound_card_explosion":"声音卡“爆炸”",
+	"item.buildingmod.sound_card_intruder_alert":"声音卡“入侵者警报”",
+	"item.buildingmod.sound_card_phaser":"声音卡“光线枪”",
+	"item.buildingmod.sound_card_red_alert":"声音卡“红色警报”",
+	"item.buildingmod.sound_card_torpedo":"声音卡“鱼雷”",
+	"item.buildingmod.sound_cart_nuke_alarm":"声音卡“核弹警报”",
+	"item.buildingmod.steel_ingot":"钢锭",
+	"item.davebuildingmod.steel_ingot":"钢锭",
+	"itemGroup.tabdaves_misc_tab":"戴夫的建材扩展：其他",
+	"itemGroup.tabdaves_mod_tab":"戴夫的建材扩展：列车相关",
+	"subtitles.arcane_quack":"奥术灌注小鸭：呱呱叫",
+	"subtitles.computer_bleep":"电脑：哔哔",
+	"subtitles.duck_quack_simple":"小橡皮鸭：呱呱叫",
+	"subtitles.emergency_exit":"",
+	"subtitles.explosion1":"爆炸",
+	"subtitles.input_ok":"输入OK",
+	"subtitles.intruder_alert":"入侵者警报：播放",
+	"subtitles.melon_quack":"建筑工葫芦小呱呱：呱呱叫",
+	"subtitles.nuclear_siren":"核弹警报：播放",
+	"subtitles.nukealert":"",
+	"subtitles.phaser_shot":"光线枪：发射",
+	"subtitles.red_alert1":"红色警报：播放",
+	"subtitles.simple_quack":"小橡皮鸭：呱呱叫",
```

```suggestion
	"subtitles.simple_quack":"呱呱！",
```
这里没写声音来源倒是。嘎嘎和呱呱你觉得那个更好

---

声音来源是与模组中的小橡皮鸭的右键交互。
这么一说感觉嘎嘎确实更好，之后会进行更改。

---

啊，我的意思是没有写来源的声音就不用写来源了，比如“脚步声”。虽然大家都知道是玩家的。

---

啊……只用看原文就好了……当然你写了也不算错